api/src/main/scala/pl/touk/nussknacker/engine/api/process/VersionId.scala

```diff
@@ -2,10 +2,9 @@ package pl.touk.nussknacker.engine.api.process
 
 import io.circe.{Decoder, Encoder}
 
-final case class VersionId(value: Long) extends AnyVal
+final case class VersionId(value: Long)
```

Comment:
Because of `CONTRIBUTIN.MD`:
- Avoid using AnyVal if the class is in API that will be used from Java